src/dal/Jalasoft.TeamUp.Resumes.DAL/SkillsApiRepository.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace Jalasoft.TeamUp.Resumes.DAL
+{
+    using System.Collections.Generic;
+    using Jalasoft.TeamUp.Resumes.DAL.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+    using Newtonsoft.Json;
+    using RestSharp;
+
+    public class SkillsApiRepository : ISkillsRepository
+    {
+        public IEnumerable<Skill> GetSkills(string name)
+        {
+            string emsiSkills = this.SkillsManager(name);
+            var response = JsonConvert.DeserializeObject<Root>(emsiSkills);
+            List<Skill> listSkills = new List<Skill>();
+            if (listSkills != null)
+            {
+                foreach (var data in response.Data)
+                {
+                    Skill skill = new Skill
+                    {
+                        Id = data.Id,
+                        Name = data.Name
+                    };
+                    listSkills.Add(skill);
+                }
+            }
+
+            return listSkills;
+        }
+
+        private IRestResponse<EmsiToken> PostEmsiToken()
+        {
+            string clientId = Constants.Constants.ClientId;
```

Comment:
use the constants directly in the line 41, not needed to assign it to a new variable